### PR TITLE
Proofreading for the Dvaered Frontier War

### PR DESCRIPTION
**Pull request by aguynamedryan**
Non-trivial changes to wording/phrasing/sentences

The author of these missions is awesome and the storyline, characters, and dialogue are very compelling.  This was a joy to edit.  I hope my suggested changes don't take anything away from the fantastic source material.

### CODE REVIEW

**Comment by aguynamedryan** [comment]
Let me rebase this for you.  How silly of me.

**Comment by not-pyroman** [comment]
It seems you made a merge conflict

**Comment by aguynamedryan** [comment]
I'm not a man of subtly and have force-pushed a rebased version of my changes

File: dat/missions/dvaered/frontier_war/fw05_triathlon.lua [hunk 1]
```diff
@@ -59,8 +59,8 @@ local mace_fail = _("Your only weapons should be Mace rockets: land and speak ag
 local joy = { _("Wohoo!"),
         _("One less!"),
         _("Baoum!"),
-        _("I'm in such a good shape!"),
-        _("Top score, I'm coming!"),
```
Comment (UncombedCoconut) [inline dat/missions/dvaered/frontier_war/fw05_triathlon.lua:63]: lol

**Comment by UncombedCoconut** [comment]
Great work, thanks!

**Pull request merged**